store/searchlayer/stop_word.go

```diff
@@ -0,0 +1,6 @@
+package searchlayer
+
+var (
+	MYSQL_STOP_WORDS = []string{"a", "about", "an", "are", "as", "at", "be", "by", "com", "de", "en", "for", "from", "how", "i", "in", "is", "it", "la", "of",
```

Comment:
In the future, we're going to split the MySQL and the Postgres layer so this is useful because we can end up with a common list of stop words to do a prefilter

Reply:
Perfect